sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -63,6 +65,117 @@ def test_input_estimator_unchanged():
     assert_true(transformer.estimator is est)
 
 
+def check_invalid_max_features(est, X, y):
```

Comment:
Often we use `check_` for helpers in tests. `test_` only applies when there are no required arguments.